samples/util/cli_args.js

```diff
@@ -62,6 +63,11 @@ function add_universal_arguments(yargs) {
             type: 'boolean',
             default: false
         })
+        .option('mqtt5', {
```

Comment:
Thought: Should we have this in it's own function that adds it, rather than the universal? Right now only the Shadow sample supports MQTT5, if I am understanding the code correctly, but with it being in the universal, all samples will have the option shown when running `--help`.